tests/test_chrono.py

```diff
@@ -0,0 +1,114 @@
+
+
+def test_chrono_system_clock():
+    from pybind11_tests import test_chrono1
+    import datetime
+
+    # Get the time from both c++ and datetime
+    date1 = test_chrono1()
+    date2 = datetime.datetime.today()
+
+    # The returned value should be a datetime
+    assert isinstance(date1, datetime.datetime)
+
+    # The numbers should vary by a very small amount (time it took to execute)
+    diff = abs(date1 - date2)
+
+    # There should never be a days/seconds difference
+    assert diff.days == 0
+    assert diff.seconds == 0
+
+    # 50 milliseconds is a very long time to execute this
+    assert diff.microseconds < 50000
```

Comment:
I changed the microseconds bit to half a second, any computer that can't execute those two lines of code within half a second has some pretty serious problems. There's also a very small chance it'll fail if the time changes between the two (due to an NTP update etc) but I feel like it's worth it to test that the results from python and c++ are the same.